src/tests/tracing/eventpipe/processinfo/processinfo.cs

```diff
@@ -78,6 +78,7 @@ public static int Main(string[] args)
             string envKey = "TESTKEY";
             string envVal = "TESTVAL";
             System.Environment.SetEnvironmentVariable(envKey, envVal);
+            System.Environment.SetEnvironmentVariable("foo", "");
```

Comment:
This will delete this env var. If you want to create an env var with no value you have to call the OS directly.